refactor ROCR documentation (#215)
ROCm/ROCR-Runtime: docs/environment_variables.rst

.. meta::
   :description: HSA runtime implementation
   :keywords: ROCR, ROCm, library, tool, runtime

.. _environment-variables:

Environment variables
========================

> The following table lists the useful environment variables.

Reviewer: The term "useful" should be removed. 
it means that other variables are useless. 

Are these the only variables? if not then say something like, 
The following table lists the most often used variables.

Author: updated.

.. list-table:: ROCR environment variables
    :header-rows: 1
    
    * - Environment Variable
      - Possible values
      - Description

    * - HSA_ENABLE_SDMA
      - 
        * 0: Disabled
        * 1: Enabled (default)

>       - This enables or disables the use of DMA engines in all copy directions (Host-to-Device, Device-to-Host, Device-to-Device) when using the

Reviewer: ```suggestion
      - Specifies the use of DMA engines in all copy directions (Host-to-Device, Device-to-Host, Device-to-Device) when using the
```

Author: @lpaoletti I guess "control" would be a better word. Updated it.

        ``hsa_memory_copy``, ``hsa_amd_memory_fill``, ``hsa_amd_memory_async_copy``, ``hsa_amd_memory_async_copy_on_engine`` APIs

    * - HSA_ENABLE_PEER_SDMA
      -
        * 0: Disabled
        * 1: Enabled (default)

>       - This enables or disables the use of DMA engines for Device-to-Device copies when using the ``hsa_memory_copy``, ``hsa_amd_memory_async_copy``, ``hsa_amd_memory_async_copy_on_engine`` APIs

Reviewer: ```suggestion
      - Specifies the use of DMA engines for Device-to-Device copies when using the ``hsa_memory_copy``, ``hsa_amd_memory_async_copy``, ``hsa_amd_memory_async_copy_on_engine`` APIs
```
